esptfa_arima/Test_Management/migrations/0013_section_adviser.py

```
@@ -0,0 +1,21 @@
+    dependencies = [
+        ('Test_Management', '0012_alter_actualposttest_options_and_more'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='section',
+            name='adviser',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)